packages/components/addon/components/hds/stepper/index.hbs

```diff
@@ -0,0 +1,3 @@
+<div role="list" class="hds-stepper" ...attributes>
```

Comment:
So I am not sure if I should leave `...attributes` here. 

Pro: a `data-` attribute could be added for tests?
Cons: Do we need it?

Also, I added the component name as a class even though we are not styling it from within the addon: 
1. it's consistent with all of the other components
2. it's a convenient styling class already attached and ready for use (by consuming teams)

Reply:
I'm +1 on leaving both of them in - leaving `...attributes` doesn't hurt - the `data-` passthrough is nice and is certainly used for tests. I think components without `...attributes` sometimes surprising to users, and I agree with your points for the class as well.

Reply:
Is there a reason for using a `role="list"` instead of a `<ol>` here?